### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by abriconi**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+import { router } from './routers';
 
 const App = () => {
 	return (
 		<div className='App'>
 			<Header></Header>
-			<Courses></Courses>
+			<RouterProvider router={router} />
+			{console.log(router)}
+			{/* <Courses></Courses> */}
```
Comment (EpamFanAnton) [inline src/App.jsx]: don't forget to remove comments and uninformative comments

File: src/components/Courses/components/CourseCard/CourseCard.tsx [hunk 2]
```diff
@@ -13,7 +14,13 @@ interface Props {
 	courseData: Course;
 }
 
-const CourseCard = ({ courseData }: Props) => {
+const CourseCard: React.FC<Props> = ({ courseData }) => {
+	const navigate = useNavigate();
+	function onClick() {
+		console.log(courseData.id);
```
Comment (EpamFanAnton) [inline src/components/Courses/components/CourseCard/CourseCard.tsx]: same here

**Review by EpamFanAnton — CHANGES_REQUESTED** [review]
While testing your app I found:
1. When first entering the app I'm not redirected to login, and just see empty page
2. If I'm not logged in and try to access `/courses` I'm being redirected to '/registration` (have to be `/login`
3. If I write the path to the course myself the app says that there is no such course (`courses/de5aaa59-90f5-4dbc-b8a9-aaf205c551ba`) after that all courses returns 'not-found'

File: src/App.jsx (hunk 1 above)
Comment (abriconi) [reply src/App.jsx]: done

File: src/components/Courses/components/CourseCard/CourseCard.tsx (hunk 2 above)
Comment (abriconi) [reply src/components/Courses/components/CourseCard/CourseCard.tsx]: done

**Review by EpamFanAnton — APPROVED** [review]
25

**Pull request merged**